test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -5448,12 +5448,13 @@ TEST_F(AtenXlaTensorTest, TestIndexAdd) {
     }
   }
   ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
-  ExpectCounterChanged("xla::index_add_", cpp_test::GetIgnoredCounters());
+  ExpectCounterChanged("xla::index_add", cpp_test::GetIgnoredCounters());
 }
 
 TEST_F(AtenXlaTensorTest, TestIndexAddInPlace) {
   int index_size = 10;
   int rank = 3;
+  double alpha = 1.0;
```

Comment:
For you case, you should use different alpha

Reply:
Consider using a `vector` of alpha values, containing, `{0, 1, <another-int>}`